README.adoc

```diff
@@ -96,13 +96,13 @@ The starter provide additional properties that can be placed in `application.pro
 
 ### Customization
 
-You can further customize the cache manager by creating beans of types `InfinispanCacheConfigurer` and/or `InfinispanGlobalConfigurer`.
```

Comment:
For symmetry reasons, I think we should also create customizers (probably both global and per cache). This way everything will be consistent.